packages/util/src/index.ts

```diff
@@ -0,0 +1,24 @@
+export * from './menu-keys'
+export * from './types'
+
+export { callAll } from './call-all'
+export { cloneElementWithRef } from './clone-element-with-ref'
+export { transparentize } from './color'
+export { combineFns } from './combine-fns'
+export { createUniversalPortal } from './create-universal-portal'
+export { debounce } from './debounce'
+export { isFunction } from './is-function'
+export { isRef } from './is-ref'
+export { generateDefaultProps, generatePropTypes } from './prop-defs'
+export { whitelistProps } from './props'
+export { shallowCompare } from './shallow-compare'
+export { capitalize } from './string'
+export { stylesFor } from './styles'
+export { useCombinedRefs } from './use-combined-refs'
+export { usePrevious } from './use-previous'
+export { useResizeObserver } from './use-resize-observer'
+export { useToggle } from './use-toggle'
+export {
+  useCloseOnDocumentEvents,
+  onGlobalEventsClose
+} from './use-close-on-document-events'
```

Comment:
i converted some of the util functions to typescript but not all. we'll need to followup later if we want everything in ts